mteb/tasks/Classification/eng/PoemSentimentClassification.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+class PoemSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="PoemSentimentClassification",
+        description="Poem Sentiment is a sentiment dataset of poem verses from Project Gutenberg.",
+        reference="https://arxiv.org/abs/2011.02686",
+        dataset={
+            "path": "poem_sentiment",
+            "revision": "329d529d875a00c47ec71954a1a96ae167584770",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2020-01-01", "2021-01-01"),
```

Comment:
This doesn't seem right? Are we sure that all of the poems are from this one year period? If not, can you come with a reasonable estimate for which years the poems can be from?

Reply:
Those are randomly picked verses from the Project Gutenberg. They could span a very wide time period (possibly starting from 1700s), so I approximated the date of collection. What do you suggest here?

Reply:
I would give a rough estimate of the date range of creation, not collection, I think that's kind of the point of the attribute

Reply:
I've updated it. But this is a rough estimate.